siliconcompiler/tools/openroad/scripts/sc_export.tcl

```diff
@@ -1,5 +1,11 @@
 # Write LEF
-write_abstract_lef "outputs/${sc_design}.lef"
+set lef_args []
+if { [lindex [dict get $sc_cfg tool $sc_tool task $sc_task {var} ord_abstract_lef_bloat_layers] 0] == "true" } {
+  lappend lef_args "-bloat_occupied_layers"
+} else {
```

Comment:
Not familiar with the semantics...
Are `-bloat_occupied_layers` and `-bloat_factor` mutually exclusive?